cime_config/testdefs/testlist_clm.xml

```diff
@@ -3379,6 +3379,26 @@
       <option name="comment">30 day exact restart test with threading for FATES fixed biogeography reduced complexity mode on an f45 grid.</option>
     </options>
   </test>
+  <test name="ERP_P128x2_Ld30" grid="f45_f45_mg37" compset="I2000Clm60FatesSpCrujraRsGs" testmods="clm/FatesColdSatPhen">
```

Comment:
Change all ctsm_sci tests to explicit crujra2024.
If need more tests, go through all implicit gswp3 tests and convert a few to explicit crujra2024 (e.g. some longer ones).

Reply:
List of implicit compsets to change to explicit crujra:
**I1850Clm60BgcCropCmip6** and delete the gswp3 compset
**I1850Clm60BgcCropCmip6waccm** and delete the gswp3 compset

Reply:
...where I find f09 tests, remember to make them f09_t232 (or whatever the correct grid name is).